src/main/java/at/ac/tuwien/kr/alpha/common/terms/Term.java

```diff
@@ -7,26 +7,22 @@
 import java.util.List;
 
 /**
- * Common representation of Terms. Terms are constructed such that each term is represented by a unique object, hence
- * term equality can be checked by object reference comparison. Each concrete subclass of a Term must implement a
- * factory-like method to obtain instances.
+ * Common representation of Terms. Terms are constructed such that each term is represented by a unique object, hence term equality can be checked by object
```

Comment:
Please configure your IDE in a way that it only auto-formats lines that are _changed_.

Reply:
Unfortunately that's something the Eclipse formatter doesn't support. However, since it would definitely be desireable to have all our formatters behave the same way, I created issue #210 to work on a possible solution using an external formatter supported by both IDEs as well as a lot of text editors.

Reply:
Currently I don´t have Eclipse installed, but last time I used it this behaviour was possible by enabling "Format edited lines" in Java > Editor > Save Actions.

Reply:
Thanks, I didn't think of that since I normally don't use save actions. Configured it now the way you suggested, hope it cuts down on useless formatting changes a bit